workshop/content/20-typescript/70-advanced-topics/200-pipelines/4000-build-stage.md

```diff
@@ -87,7 +87,7 @@ export class WorkshopPipelineStack extends cdk.Stack {
 }
 {{</highlight>}}
 
-This imports and creates an instance of the `WorkshopPipelineStage`. Later, you might instantiate this stage multiple times (e.g. you want a Production deployment and a separate devlopment/test deployment).
+This imports and creates an instance of the `WorkshopPipelineStage`. Later, you might instantiate this stage multiple times (e.g. you want a Production deployment and a separate development/test deployment).
```

Comment:
We will also need this fix in the Java and Python workshops. the .NET workshop doesn't have this error.